site/zenodo_rdm/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.0'
```

Comment:
Added, otherwise test for version was failing

Reply:
Maybe this is a question for @zzacharo, if the package is only accessible inside the instance (no pypi) and is installed with editable. what is the idea behind having a version string? i.e. the version string will never be used for installing since we wont be able to install previous versions (not packaged/published), the only possibility is VCS.

Reply:
I think that it started as an idea that you can publish this initially if the user wants but maybe is better to remove it and hardcode a version in the `setup.cfg`. @alejandromumo can you delete this file + the test? We will need to make the same changes in the `cookiecutter-invenio-rdm`

Reply:
@zzacharo we can leave in cookiecutter, I do see that it can be useful... However, I'd remove it from this instance.